pom.xml

```diff
@@ -8,18 +8,37 @@
 
 	<properties>
 		<java-version>1.7</java-version>
+
+		<apache.oltu.version>1.0.2</apache.oltu.version>
+		<slf4j.version>1.7.25</slf4j.version>
 	</properties>
 
 	<dependencies>
 		<dependency>
 			<groupId>org.json</groupId>
 			<artifactId>json</artifactId>
-			<version>20141113</version>
+			<version>20180130</version>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.oltu.oauth2</groupId>
 			<artifactId>org.apache.oltu.oauth2.client</artifactId>
-			<version>1.0.0</version>
+			<version>${apache.oltu.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.oltu.oauth2</groupId>
+			<artifactId>org.apache.oltu.oauth2.common</artifactId>
```

Comment:
Did I get this right? ...mentioning this is just to exclude slf4j?

Reply:
The thing is here that oltu has slf4j as dependency but with an old version.
I wanted to use the latest version of the 1.7 branch, so i exclude it here and add it as a separate dependency later on...